### PR TITLE
[#12] UI Red Button

### PR DESCRIPTION
**Pull request by ghost**
Hi,
can You pls check my code?

### CODE REVIEW

File: styles/App.css [hunk 1]
```diff
@@ -1,3 +1,22 @@
...
+
+.button--red {
+    background: rgba(255, 0, 0, 0.8);
+    box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    color: #fff;
+}
+
+.button--uppercase {
+    text-transform: uppercase;
+}
```
Comment (Ryczko) [inline styles/App.css:22]: I suggest moving these styles to one file with styles of this component and importing it here, as someone did in the component that is already on the master

**Review by GoraJakub — APPROVED** [review]
Looks good to me.

File: styles/App.css (hunk 1 above)
Comment (Ryczko) [inline styles/App.css:22]: I suggest moving these styles to one file with styles of this component and importing it here, as someone did in the component that is already on the master

Comment (GoraJakub) [reply styles/App.css:22]: I was thinking the same, even though there's no information provided as it is e.g. in [#7]

Comment (domq077) [reply styles/App.css:22]: I did it in this way, but then i changed. I thought that it is general style and can be use it in other part of application that i pushed this to app.css file. But if team want i can change this to my previous version.

Comment (Ryczko) [reply styles/App.css:22]: personally, I don't mind

**Pull request merged**